config/locales/en/notifiers/course/assessment/answer/comment_notifier/replied/user_notifications/email.yml

```diff
@@ -0,0 +1,15 @@
+en:
+  notifiers:
+    course:
+      assessment:
+        answer:
+          comment_notifier:
+            replied:
+              user_notifications:
+                email:
+                  subject: '%{course} New comment on %{topic}'
+                  message: >
+                    %{post_author} commented, %{post}
```

Comment:
Including our HTML helper into the mailer messes up the hyperlink HTML code.

Using the helper and `format_html` makes the message look like it got escaped twice.

Rail's `simple_format` takes care of the `script` tag already.